### ISSUE TITLE
Gotype fails to import subpackages of the same project

### ISSUE THREAD

**Issue by obi1kenobi** [sirreal/SublimeLinter-contrib-gotype#3]
Say you have a project that is organized as follows:

```
$GOPATH/
+src/
 +name/
  +a/
   +a.go
  +b/
   +b.go
```

It is perfectly legal for `b.go` to have a line such as `import name/a`. However, in the current setup, `gotype` fails to do the import (presumably because of the way files are copied to the temp directory). As a result, I get a ton of lint errors on everything that uses that package, even though the code will run just fine.

I am currently forced to run `gotype` with the following `ignore_match` settings:

```
"could not import [a-z/]* \\(can't find import: [a-z/]*\\)",
"undeclared name: [a-zA-Z_]*"
```

I was wondering if there is something I am doing wrong, or if this is a bug in the plugin.

**Comment by MOZGIII** [comment]
That's because this plugin copies files over to the temp dir. Why would you do that?

**Comment by zj8487** [comment]
any news on the bug?

**Comment by sirreal** [comment]
@zj8487 This issue contains all the news on this bug.

If anyone would like to fix this, or help out with this plugin in a permanent way, that would be more than welcome.

To be completely transparent, this is a linting plugin for a language I don't use for an editor I don't use. It's extremely unlikely I will be working on this further.

**Comment by rasky** [comment]
The main issue of this bug can't be fixed; if a GOPATH isn't defined, there's nothing the plugin can do. You can also define one in the plugin configuration, if you want (there's a configuration key `gopath` for that).

The other issue (running `gotype` on the full package) is fixed in 1.2.0, just released.
